deploy.sh

```diff
@@ -3,6 +3,7 @@
 
 # Environment variables
 APP_NAME="news-search-api"
+API_REPLICAS=8
 API_PORT_BASE=8000
 UI_PORT_BASE=8501
 IMAGE_TAG="latest"
```

Comment:
If we're defaulting to `dev` configs e.g. IMAGE_TAG="latest", then also default to API_REPLICAS expected in `dev`.